helm/alfresco-content-services/Chart.yaml

```diff
@@ -3,7 +3,7 @@
 # Request an extended 30-day trial at https://www.alfresco.com/platform/content-services-ecm/trial/docker
 apiVersion: v1
 name: alfresco-content-services
-version: 5.0.0
+version: 5.0.0-115
```

Comment:
-115 suffix is still present

Reply:
Yes because I don't want to overwrite the incubator chart until the review is done.

Reply:
OK, gotcha.